docs/src/components/home/UserFeedbacks.tsx

```diff
@@ -120,59 +85,104 @@ const TESTIMONIALS = [
-              aria-label={`Testimonial from ${item.profile.name}`}
-              onClick={() => setSlideIndex(index)}
-              sx={{
-                display: 'inline-block',
-                width: 16,
-                height: 16,
-                borderRadius: '50%',
-                p: '4px',
-                ml: index !== 0 ? '2px' : 0,
-                '&:focus': {
-                  boxShadow: (theme) => `0px 0px 0px 2px ${theme.palette.primaryDark[400]}`,
-                },
-              }}
-            >
-              <Box
-                sx={{
-                  height: '100%',
-                  borderRadius: 1,
-                  bgcolor: 'primaryDark.500',
-                  ...(index === slideIndex && {
-                    bgcolor: 'primaryDark.300',
-                  }),
-                }}
-              />
-            </ButtonBase>
-          ))}
+    <Box
+      sx={{
+        p: 3,
+        height: '100%',
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'space-between',
+        color: '#FFF',
+      }}
+    >
+      <Typography
+        sx={{
+          mb: 2.5,
+          lineHeight: 1.6,
+          color: 'grey.200',
+          fontSize: (theme) => theme.typography.pxToRem(15),
+        }}
+      >
+        {quote}
+      </Typography>
+      <Box sx={{ display: 'flex', alignItems: 'center', gap: 1.5 }}>
+        <Box
+          sx={(theme) => ({
+            p: 0.5,
+            border: '1px solid',
+            borderColor: 'primary.800',
+            bgcolor: alpha(theme.palette.primary[900], 0.5),
+            borderRadius: 99,
+          })}
+        >
+          <Avatar
+            srcSet={profile.avatarSrcSet}
+            src={profile.avatarSrc}
+            alt={`${profile.name}'s profile picture`}
+            imgProps={{ loading: 'lazy' }}
+            sx={{
+              width: 36,
+              height: 36,
+            }}
+          />
         </Box>
+        <div>
+          <Typography variant="body2" fontWeight="semiBold" color="text.primary">
+            {profile.name}
+          </Typography>
+          <Typography variant="body2" color="text.secondary">
+            {profile.role}
+          </Typography>
+        </div>
+        <Box sx={{ ml: 'auto' }}>{profile.company}</Box>
       </Box>
-      <SwipeableViews index={slideIndex} onChangeIndex={(index) => setSlideIndex(index)}>
-        {TESTIMONIALS.map((item) => (
-          <Feedback key={item.profile.name} {...item} />
-        ))}
-      </SwipeableViews>
     </Box>
   );
 }
+
+export default function UserFeedbacks() {
+  return (
+    <Grid
+      container
+      sx={(theme) => ({
+        mt: 4,
+        backgroundColor: 'rgba(255,255,255,0.01)',
+        border: '1px solid',
+        borderColor: 'divider',
+        borderRadius: 1,
+        overflow: 'clip',
+        '> :nth-of-type(1)': { borderBottom: `1px solid ${theme.palette.primaryDark[700]}` },
+        '> :nth-of-type(2)': {
```

Comment:
From a syntax perspective, I would be more in favor to always be explicit. A bit similar to the native CSS feature https://developer.mozilla.org/en-US/docs/Web/CSS/Nesting_selector (not directly related API though)

```suggestion
        '& > :nth-of-type(2)': {
```